src/help/getRenderProperties.js

```diff
@@ -52,12 +52,18 @@ function getRenderProperties(element){
 		};
 	}
 	// If canvas (in node)
-	else if(element.getContext){
+	else if(element && element.getContext){
 		return {
 			element: element,
 			renderer: renderers.CanvasRenderer
 		};
 	}
+	else if(element && typeof element === 'object') {
```

Comment:
You are right, this does accually have to be fixed. I though that DOM elements returned an other type which it does not. This is the minimum thing that to check that is not a DOM element with something like this http://stackoverflow.com/a/4767731/838789.

But an other solution that for example checks for an empty element might be as good or better